convert/src/label_studio_to_badgerdoc/labelstudio_format/label_studio_format.py

```diff
@@ -68,6 +108,17 @@ def convert_annotation_from_bd(
                     labels=[obj.category],
                 ),
             )
+            # taxon_value = [taxon_attr['value'] for taxon_attr in obj.data.dataAttributes]
+            # item.value.taxons = [taxon_value[0] if taxon_value else None for label in item.value.labels]
+            if obj.data.dataAttributes:
+                item.value.taxons = [
+                    obj.data.dataAttributes[0]["value"],
+                ]
+            else:
+                item.value.taxons = [
+                    None,
+                ]
```

Comment:
List with single element [None] is weird (also wrong linter formatting due to redundant comma).
You can use your "safe_list_get_item" function without `if/else` here like:
```
item.value.taxons = [
    safe_list_get_item(my_list=obj.data.dataAttributes, index=0, default={}).get("value")
]
```

Reply:
I would prefer to keep it this way.
Right now shows the established format of "taxons" field in label studio format files